requirements.txt

```diff
@@ -8,3 +8,4 @@ socketIO-client == 0.5.6
 stopit >= 1.1.1, < 1.1.999
 yamlconf >= 0.0.3, < 0.0.999
 revscoring >= 0.7.10
+redis
```

Comment:
Did you see the commit message?

> Not sure if this is a hard dependency, but the test config seems to want it.

So is the correct patch to set the default config to no cache at all?